examples/abitest/client/grpc_test_server.cc

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2020 The Project Oak Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "examples/abitest/client/grpc_test_server.h"
+
+#include "asylo/util/logging.h"
+
+using ::oak::examples::abitest::GrpcTestRequest;
+using ::oak::examples::abitest::GrpcTestResponse;
+
+namespace oak {
+namespace test {
+
+grpc::Status GrpcTestServer::RunTests(grpc::ServerContext* context,
+                                      const oak::examples::abitest::ABITestRequest* req,
+                                      oak::examples::abitest::ABITestResponse* rsp) {
+  // Test server only implements the gRPC test methods.
+  return grpc::Status(grpc::StatusCode::UNIMPLEMENTED, "");
+}
+
+grpc::Status GrpcTestServer::UnaryMethod(grpc::ServerContext* context, const GrpcTestRequest* req,
+                                         GrpcTestResponse* rsp) {
+  if (req->method_result_case() == GrpcTestRequest::kErrCode) {
```

Comment:
(optional) Is it worth making this a switch? May or may not help with readability, clarifying the intent.

Reply:
I prefer the current way around, particularly across the different method types: any time there's an error requested, immediately `return` with that error.